src/server/views/dashboard/lists.njk

```diff
@@ -26,21 +26,21 @@
         {% for list in lists %}
           <tr class="govuk-table__row">
             <td scope="row" class="govuk-table__header">
-              {% if userIsListValidator(req, list) or userIsListPublisher(req, list) %}
+              {% if userIsListPublisher(req, list) or req.user.isSuperAdmin() %}
                 <a href="{{ dashboardRoutes.listsItems.replace(':listId', list.id) }}" class="govuk-link">{{ _.startCase(list.type) }}</a>
               {% else %}
                 {{ _.startCase(list.type) }}
               {% endif %}
             </td>
             <td class="govuk-table__cell">{{ list.country.name }}</td>
-            
+
             <td class="govuk-table__cell govuk-table__cell--numeric"></td>
 
             <td class="govuk-table__cell govuk-table__cell--numeric">
-              {% if userIsListAdministrator(req, list) %}
+              {% if userIsListPublisher(req, list) or req.user.isSuperAdmin() %}
```

Comment:
can we also add .isListPublisher() to req.user? so they're called the same way?

Reply:
FYI, this point is also mentioned above @RichardBray.